glTF-BinExporter/RhinoDocGltfConverter.cs

```diff
@@ -94,7 +98,7 @@ public Gltf ConvertToGltf()
 
                 int nodeIndex = dummy.Nodes.AddAndReturnIndex(node);
 
-                if(options.ExportLayers)
+                if (options.ExportLayers)
                 {
                     AddToLayer(RhinoDoc.ActiveDoc.Layers[exportData.Object.Attributes.LayerIndex], nodeIndex);
```

Comment:
We need to remove these ActiveDoc calls since they aren't safe and could crash on Mac. I missed this one in the last PR. I've added a commit to properly deliver the RhinoDoc to the RhinoDocGltfConverter. Use that instead of these ActiveDoc calls.

Reply:
c879f6c0e12125a5f1b34176d1cfe80bdb12aa8a